src/main/scala/org/broadinstitute/dsde/firecloud/webservice/Ga4ghApiService.scala

```diff
@@ -18,7 +18,7 @@ trait Ga4ghApiService extends HttpService with FireCloudDirectives {
       pathPrefix("tools") {
         path (Segment / "versions" / Segment / Segment / "descriptor") { (id, versionId, descriptorType) =>
           get {
-            val targetUri = Uri(s"$agora/ga4gh/tools/$id/versions/$versionId/$descriptorType/descriptor")
+            val targetUri = Uri(s"$agora/ga4gh/v1/tools/$id/versions/$versionId/$descriptorType/descriptor")
```

Comment:
please move orch's api to /ga4gh/v1 too! Yes this is technically a breaking API change, but we warned people during the demo, and we know that nobody is using it yet - it hasn't even been released yet.